src/components/UserInfo.vue

```diff
@@ -1,24 +1,42 @@
+<!--suppress XmlInvalidId -->
 <template>
-  <div id="user-info">
-    <v-alert :value="!!successMessage" type="success" dismissible>{{ successMessage }}</v-alert>
-    <v-alert :value="!!errorMessage" type="error" dismissible>{{ errorMessage }}</v-alert>
+  <div id="user-info" v-if="dataReady">
```

Comment:
I use `v-if="dataReady"` to prevent the component from rendering until the data loads. This makes things look a little nicer: it doesn't render empty fields followed by data "popping in".